ValidationLibrary.AzureFunctions.Tests/StatusEndpointTests.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using NUnit.Framework;
+using Octokit;
+using ValidationLibrary.Rules;
+
+namespace ValidationLibrary.AzureFunctions.Tests
+{
+    [TestFixture]
+    public class StatusEndpointTests
+    {
+        private StatusEndpoint _statusEndpoint;
+
+        [Test]
+        public void Run_NormalStatusEndpointCheck()
+        {
+            Environment.SetEnvironmentVariable("GitHub:Organization", "mock");
+            Environment.SetEnvironmentVariable("GitHub:Token", "mock");
+            
+            var ruleType = typeof(HasLicenseRule);
+            var expectedRulesTypes = ruleType.Assembly.GetExportedTypes().Where(t => t.GetInterface(nameof(IValidationRule)) != null && !t.IsAbstract);
+            var expectedRules = expectedRulesTypes.Select(r =>
```

Comment:
Here is an example of mocking `IValidationRule` for our purposes.
```
var rule = Substitute.For<IValidationRule>();
rule.GetConfiguration().Returns(new Dictionary<string,string>());
```
 I assume you can change following example to actually returns those configuration values we wish.